src/cli/move.ts

```diff
@@ -106,6 +106,114 @@ export class Move {
     return this.runCommand(cliArgs);
   }
 
+  /**
+   * Function to create a new object and publish the Move package to it on the Aptos blockchain
+   *
+   * @param args.packageDirectoryPath Path to a move package (the folder with a Move.toml file)
+   * @param args.addressName Address name for the Move package
+   * @example
```

Comment:
example is wrong

Reply:
this indeed should be the name instead of value, here's the shell script version

```sh
aptos move create-object-and-publish-package \
  --package-dir $MOVE_DIR \
  --address-name launchpad_addr \
  --named-addresses "launchpad_addr=$PUBLISHER_ADDR,initial_creator_addr=0x7fe2b067b33642d6e5fea417452abeb131bb80ea07ad5d2ff0a0f58292238393"\
  --profile $PUBLISHER_PROFILE \
  --assume-yes

```

Reply:
i see. a bit confusing as it is now. can we get 1 example that shows what arguments to pass?
```
{
 packageDirectoryPath:"some/path/to/move.mv;
 addressName: "mode_address_name";
 namedAddresses: {
     alice:0x1234, bob:0x5678
    }
}
```

Reply:
included a complete example